Makefile

```
@@ -211,6 +214,17 @@ test.integration.postgres:
 		-coverprofile=coverage.postgres.out \
 		./test/integration
 
+.PHONY: test.integration.enterprise.postgres
+test.integration.enterprise.postgres:
+	@./scripts/check-container-environment.sh
+	@TEST_DATABASE_MODE="postgres" TEST_ENTERPRISE="on" GOFLAGS="-tags=integration_tests" go test -v -race \
```

I don't think you'll be able to include the `--race` flag right now due to a known bug:

https://github.com/Kong/kubernetes-ingress-controller/issues/1324

We need to fix this, but I don't think we need to do it in the scope of _this_ PR.

```suggestion
	@TEST_DATABASE_MODE="postgres" TEST_ENTERPRISE="on" GOFLAGS="-tags=integration_tests" go test -v \
```

---

The PR fixed one "data race" issue (#1825), keep the flag on helps to ensure no regression on enterprise postgress deployment.